Makefile

```diff
@@ -104,10 +108,13 @@ update-minimal: build-minimal
 	GIT_REVISION=${GIT_REVISION} docker build --network=host --build-arg GIT_REVISION \
 		-f Dockerfile.assisted-service . -t $(SERVICE)
 
-update-minikube: build
+update-minikube: build build-dummy-ignition-image
```

Comment:
it should be in `jenkins-deploy-for-subsystem` as well

Reply:
fixing